src/mcts/search.cc

```diff
@@ -989,7 +990,8 @@ bool SearchWorker::AddNodeToComputation(Node* node, bool add_if_cached) {
 
   if (node && node->HasChildren()) {
     // Legal moves are known, use them.
-    for (auto edge : node->Edges()) {
+    moves.reserve(node->GetNumEdges());
+    for (const auto& edge : node->Edges()) {
       moves.emplace_back(edge.GetMove().as_nn_index());
     }
   } else {
```

Comment:
There is another reserve() a few lines later that also doesn't seem to help.

Reply:
Again, I was able to measure an improvement on my low end laptop but not on my high end desktop.

Reply:
Here it's a bit more plausible than with applying Dirichlet noise, but as it seems that there's something wrong with your measurements (they should have not shown anything there), it would be better if someone else retested/reconfirmed it. Or at least if you could post numbers from your tests.

In general it's good idea only to optimize where it gives benefits, adding reserve() line just for the case doesn't do any good.